pkgs/development/python-modules/elasticsearch-curator/default.nix

```diff
@@ -45,6 +45,13 @@ buildPythonPackage rec {
     funcsigs
   ];
 
+  postPatch = ''
+    substituteInPlace setup.cfg \
+      --replace 'click>=6.7,<7.0' 'click'
+    substituteInPlace setup.py \
+      --replace 'click>=6.7,<7.0' 'click'
```

Comment:
You're sure both of these need to be patched?

Reply:
Yep!